pipeline/tools/luigi-service.yaml

```diff
@@ -1,12 +1,10 @@
 installed: false
-dependencies:
-service-file: &svcfile !get_tool_path "{luigid}"
+project-dir: &proj !get_default "{project-dir}"
+service-file: &svcfile !join_path [*proj, luigid.service]
 
 commands:
 - !join [sudo cp, *svcfile, /lib/systemd/system/luigid.service]
 - !join [sudo cp, *svcfile, $(which luigid), /usr/local/bin]
 - sudo systemctl daemon-reload
 - sudo systemctl start luigid.service
 - sudo systemctl enable luigid.service
-
```

Comment:
I missed this one, probably even said it wasn't necessary.  However, I missed the `$(which luigid)` which is another pattern that makes `shell: true` a necessity.